packages/workbox-core/.npmignore

```diff
@@ -0,0 +1 @@
+browser.mjs
```

Comment:
Exactly. We can use an unfriendly file name for this like `._browser.mjs` which npm will exclude by default, but I think this in cleaner.

That being said, I'm quickly leaning on auto-generating this file so we can ensure public classes / files are exported in the expected way.